classes/notification/managerDelegate/PendingRevisionsNotificationManager.inc.php

```diff
@@ -43,8 +43,8 @@ public function getNotificationUrl($request, $notification) {
 		$stageData = $this->_getStageDataByType();
 		$operation = $stageData['path'];
 
-		import('lib.pkp.controllers.grid.submissions.SubmissionsListGridCellProvider');
-		return SubmissionsListGridCellProvider::getUrlByUserRoles($request, $submission, $notification->getUserId(), $stageData['path']);
+		import('classes.core.ServicesContainer');
+		return ServicesContainer::instance()->getWorklowUrlByUserRoles($submission, $notification->getUserId(), $stageData['path']);
```

Comment:
I believe this should be `ServicesContainer::instance()->get(‘submission’)->getWorklowUrlByUserRoles()`